package.json

```diff
@@ -91,14 +91,15 @@
     "body-parser": "^1.16.1",
     "bootstrap": "^3.3.7",
     "boron": "^0.2.3",
+    "config": "^1.25.1",
```

Comment:
Yes, it is used here https://github.com/MakingSense/mern-seed/pull/17/files#diff-ce90ba09abeb73becc09bc91a8b8c04fR2.

No, it shouldn't be a *dev* dependency, it is used in the server side code (api).